tests/vmi_multus_test.go

```diff
@@ -595,6 +595,21 @@ var _ = Describe("[Serial]SRIOV", func() {
 		sriovResourceName = "openshift.io/sriov_net"
 	}
 
+	createNetwork := func(networkName string, namespace string, networkAttachmentDefinition string) error {
+		return virtClient.RestClient().
+			Post().
+			RequestURI(fmt.Sprintf(postUrl, namespace, networkName)).
+			Body([]byte(fmt.Sprintf(networkAttachmentDefinition, networkName, namespace, sriovResourceName))).
+			Do().Error()
+	}
+
+	deleteNetwork := func(networkName, namespace string) error {
```

Comment:
I do not see it used in this commit, so please define it where it is used (or replace the needed in this commit).